openrtb_ext/bidders.go

```diff
@@ -564,10 +564,42 @@ type BidderParamValidator interface {
 	Schema(name BidderName) string
 }
 
+type paramsValidatorHelper interface {
+	readDir(name string) ([]os.DirEntry, error)
+	readFile(name string) ([]byte, error)
+	newReferenceLoader(source string) gojsonschema.JSONLoader
+	newSchema(l gojsonschema.JSONLoader) (*gojsonschema.Schema, error)
+	abs(path string) (string, error)
+}
+
+type paramsHelper struct{}
+
+func (paramsHelper) readDir(name string) ([]os.DirEntry, error) {
+	return os.ReadDir(name)
+}
+
+func (paramsHelper) readFile(name string) ([]byte, error) {
+	return os.ReadFile(name)
+}
+
+func (paramsHelper) newReferenceLoader(source string) gojsonschema.JSONLoader {
+	return gojsonschema.NewReferenceLoader(source)
+}
+
+func (paramsHelper) newSchema(l gojsonschema.JSONLoader) (*gojsonschema.Schema, error) {
+	return gojsonschema.NewSchema(l)
+}
+
+func (paramsHelper) abs(path string) (string, error) {
+	return filepath.Abs(path)
+}
```

Comment:
Do we need to mock `abs()`. In unit tests, we could setup `fsSet` map for abs path

Reply:
We do need to mock abs() function here because we need to test a case where `abs` function returns an error. We won't be able to test this by setting fsSet map as their is a check where {bidder-name}.json file must be defined because making a call to `abs` function in `NewBidderParamsValidator`.

Reply:
It's fine, but we generally don't strive for 100% test coverage.